.storybook/pages/ProjectOverview/ProjectOverview.tsx

```diff
@@ -18,33 +19,53 @@ export const ProjectOverview: React.FC = () => (
-        <Hr />
-        <div className="fpo">
-          <div>Section title with image in front (Power Focus Areas)</div>
-          <div className="fpo">Tooltip (up for discussion)</div>
-          <div className="fpo">Hoverable/Linkable Card Recipe (naming TBD)</div>
-        </div>
-        <div className="fpo">
-          <div>Section title with image in front (Additional Focus Areas)</div>
-          <div className="fpo">
-            <div>Hoverable/Linkable Card Recipe (naming TBD)</div>
-            <div className="fpo">Badge? Status badge?</div>
-          </div>
-          <div className="fpo">
-            <div>Hoverable/Linkable Card Recipe (naming TBD)</div>
-            <div className="fpo">Badge? Status badge?</div>
-          </div>
-        </div>
-        <div className="fpo">
-          <div>Cognitive skills (smaller section title than ones above)</div>
-          <div className="fpo">Text passage with links?</div>
-        </div>
+        <Heading className={utilityStyles['u-margin-bottom-lg']} as="h2" size={1}>What is this Project About?</Heading>
+        <TextPassage size="lg">Consectetur adipiscing elit. Ac id velit ut egestas arcu. At maecenas urna, risus donec praesent eu consectetur. Nunc non eu mattis sem turpis id dictum. Volutpat ornare turpis ultrices augue bibendum pellentesque. Habitasse.</TextPassage>
+       <Button className={utilityStyles['u-margin-bottom-lg']} text="See More" variant="link" />
+       <Heading className={utilityStyles['u-margin-bottom-sm']} as="h3" size={4}>Essential Questions</Heading>
+        <TextPassage className={utilityStyles['u-margin-bottom-lg']}>
+					<ul>
+						<li>How are structures in the body organized?</li>
+						<li>How do these structures work together to support life? But really need to see what it looks like if we have an extra long question.</li>
+						<li>How are structures in the body organized?</li>
+					</ul>
+					</TextPassage>
+				<Heading className={utilityStyles['u-margin-bottom-sm']} as="h3" size={4}>Key Take Aways</Heading>
+				<TextPassage className={utilityStyles['u-margin-bottom-lg']}>
+					<ol>
+						<li>Groups of specialized cells make up organs and body systems, which work together to support an organism’s survival. </li>
+						<li>How are structures in the body organized? More text blah blah blah.</li>
+						<li>How are structures in the body organized?</li>
+					</ol>
+					</TextPassage>
+        <Hr className={utilityStyles['u-margin-bottom-none']}  />
+        <Section titleBefore={<img style={{ display: 'block', borderRadius: '50%'}} src="https://via.placeholder.com/32x32" alt="placeholder" />} title="Power Focus Areas">
+          <Card href="#">
+						<CardBody>
+							<Heading className={utilityStyles['u-margin-bottom-sm']} as="h3" size={4}>What Was Medieval Japan Like?</Heading>
+							<TextPassage>In this Focus Area, you explore the main question: How does the structures of organisms enable life's functions?</TextPassage>
```

Comment:
looks like lint is unhappy about these single quotes https://github.com/chanzuckerberg/edu-design-system/runs/5741773933?check_suite_focus=true

you could disable the rule for this whole file